cider-dap/src/adapter.rs

```diff
@@ -1,12 +1,74 @@
+use dap::types::{Breakpoint, Source, SourceBreakpoint, Thread};
 use std::fs::File;
 pub struct MyAdapter {
     #[allow(dead_code)]
     file: File,
-    // Other fields of the struct
+    breakpoints: Vec<(Source, i64)>,
```

Comment:
Can you add a comment for `breakpoints` and `threads` to mark that it is a placeholder?